app/jenkins-for-jira-ui/src/components/JenkinsServerList/ConnectedServer/ConnectedServers.tsx

```diff
@@ -14,19 +14,18 @@ import moment from 'moment';
 
 import { StyledButtonContainerConnectedServers } from '../../ConnectJenkins/ConnectJenkins.styles';
 import { JenkinsIcon } from '../../icons/JenkinsIcon';
-import {
-	StyledConnectedServerTableHeaderTitleContainer,
-	StyledConnectedServerTableCellIconContainer,
-	StyledConnectedServerTableHeaderContainer,
-	StyledConnectedServerTableContainer,
-	StyledConnectedServerTableHeaderTitle, waitingForDeploymentText
-} from './ConnectedServers.styles';
 import {
 	StyledConnectedServerContainer,
 	StyledConnectedServerLatestEventIconContainer,
+	StyledConnectedServerTableHeaderTitleContainer,
```

Comment:
This is an unrelated change but there was a duplicate import warning that kept popping up. Figured this PR is small enough that I could resolve that here.